### PR TITLE
Renamed Pages and updating Details Page to fetch data from DB

### PR DESCRIPTION
**Pull request by twentymurial33**
Renamed the pages to actually describe what the user should be expecting. Landing page is where the user originally goes to and will be routed to the details page - each clicked item routes to the details page.

image

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-  font-size: 1.5em;
-  text-align: center;
-  color: white;
-`;
+import Container from "@mui/material/Container";
 
 function Header() {
   return (
-    <AppBar position="static">
+    <AppBar postion="static">
```
Comment (brentmclark) [inline client/src/components/Header.js:8]:
```suggestion
    <AppBar position="static">
```

File: client/src/components/Header.js [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
       <Container maxWidth="xl">
-        <h1>Murial's Restaurant</h1>
+        <h1 style={{ textAlign: "center" }}>Murial's Restaurant</h1>
```
Comment (brentmclark) [inline client/src/components/Header.js:10]: Let's talk about utility classes in our next session.  They can be used to avoid online styles entirely.

**Review by brentmclark — APPROVED** [review]
Looks good, but a few quick touch ups 

I see you went with style objects instead of CSS syntax.  How are you liking that?

File: client/src/components/Header.js (hunk 2 above)
Comment (twentymurial33) [reply client/src/components/Header.js:10]: Sounds good

**Pull request merged**